### ISSUE TITLE
fatal error when adding authorization.tableHandler in the config

### ISSUE THREAD

**Issue by faFrafa** [mevdschee/php-crud-api#869]
Hi all,
congratulations for the API, it's really useful.  I am facing a fatal error when using authorization.tableHandler. 

When I remove the authorization.tableHandler from my config, I get no error from the REST call. 

But when I add it (as shown below), I get this error:
<b>Fatal error</b>:  Constant expression contains invalid operations in .........api.php</b> on line <b>**11563**</b><br />

The line 11563 corresponds to the "private $values ..." line.

See below:
    class Config
    {
        private $values = [
            'driver' => null,
            'address' => null,
            'port' => null,
            'username' => '',
            'password' => '',
            'database' => '',
            'tables' => '',
            'middlewares' => 'cors,errors,json,dbAuth,authorization',
	     'dbAuth.registerUser' => '1',
	     'dbAuth.passwordLength' => '8',
	     'authorization.tableHandler' => function ($operation, $tableName) {
        		return !($tableName == 'users');
	      },
	      'controllers' => 'records,geojson,openapi,status',
         .............

What's wrong with the code?

Thank you in advance,
Fausto

**Comment by mevdschee** [comment]
>Constant expression contains invalid operations

This means you need to set the private variable `$values` in the constructor of the Config class, not in the definition of the variable.

**Comment by faFrafa** [comment]
Thank you for the quick answer. I was using the wrong place, because I found 'middlewares' in the private $values array, I thought it was the right place to place the 'authorization' middleware. 

Reading the code I saw that $config and $values are merged in the __construct and understood that all the config parameters should go under the $config array.
 
Thank you for pointing me on the right direction.
Have a nice w.e.
